components/org.wso2.bfsi.identity.extensions/src/test/java/org/wso2/bfsi/identity/extensions/validator/resources/ValidatorTestDataProvider.java

```diff
@@ -0,0 +1,157 @@
+
+import org.testng.annotations.DataProvider;
+
+/**
+ * validator test data provider resource.
+ */
+public class ValidatorTestDataProvider {
+
+    private String claimsTemplate = "{\n" +
+            "  \"aud\": \"https://localhost:9443/oauth2/token\",\n" +
+            "  \"response_type\": \"code id_token\",\n" +
+            "  \"client_id\": \"iTKOfuqz46Y1HVY2BF0Z7JM18Awa\",\n" +
+            "  \"redirect_uri\": \"https://localhost/test/a/app1/callback\",\n" +
+            "  \"scope\": \"${SCOPE}\",\n" +
+            "  \"state\": \"af0ifjsldkj\",\n" +
+            "  \"nonce\": \"n-0S6_WzA2Mj\",\n" +
+            "  \"claims\": {\n" +
+            "    \"sharing_duration\": \"7200\",\n" +
+            "    \"id_token\": {\n" +
+            "      \"acr\": {\n" +
+            "        \"essential\": true,\n" +
+            "        \"values\": [\n" +
+            "          \"urn:cds.au:cdr:3\"\n" +
+            "        ]\n" +
+            "      }\n" +
+            "    },\n" +
+            "    \"userinfo\": {\n" +
+            "      \"given_name\": null,\n" +
+            "      \"family_name\": null\n" +
+            "    }\n" +
+            "  }\n" +
+            "}";
+
+
+    @DataProvider(name = "dp-checkValidScopeFormat")
+    public Object[][] dpCheckValidScopeFormat() {
+
+        return new Object[][]{
+                {claimsTemplate.replace("${SCOPE}",
+                "openid bank:accounts.basic:read bank:accounts.detail:read bank:transactions:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "  openid bank:accounts.basic:read  bank:accounts.detail:read bank:transactions:read  ")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid profile bank:accounts.basic:read bank:accounts.detail:read bank:transactions:read")}
+        };
+    }
+
+    @DataProvider(name = "dp-checkValidSingleScopes")
+    public Object[][] dpCheckValidSingleScopes() {
+
+        return new Object[][]{
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.detail:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:transactions:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:payees:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:regular_payments:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid common:customer.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid common:customer.detail:read")},
+
+                //
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/13/commits/cd47ead8b0ab89e7914625db1aa8437e57dc4655